primefaces-showcase/src/main/webapp/ui/misc/outputLabel.xhtml

```diff
@@ -20,39 +20,25 @@
         <div class="card">
             <h:form>
                 <p:messages id="messages"/>
-                <h:panelGrid columns="2" cellpadding="7">
-                    <h:outputLabel for="name" value="Standard Label"/>
-                    <p:inputText id="name" required="true"/>
-
-                    <p:outputLabel for="@next" value="Extended Label"/>
-                    <p:inputText id="extended" required="true"/>
-
-                    <p:outputLabel for="@next" value="Number"/>
-                    <p:spinner id="number" value="0"/>
-
-                    <p:commandButton value="Submit" update="@form" styleClass="mt-3"/>
-                    <p:spacer />
-
-                    <!-- For testing, don't remove!
```

Comment:
Thanks I think this were safe to remove as I added Integration Tests for all of these scenarios.

Reply:
Yeah, I checked